crates/primitives/src/chain/spec.rs

```diff
@@ -778,133 +789,6 @@ mod tests {
         );
     }
 
-    /// Checks that time-based forks work
-    ///
-    /// This is based off of the test vectors here: https://github.com/ethereum/go-ethereum/blob/5c8cc10d1e05c23ff1108022f4150749e73c0ca1/core/forkid/forkid_test.go#L155-L188
-    #[test]
-    fn timestamped_forks() {
```

Comment:
@onbjerg what's the reason behind removing this test?

Reply:
Described in 41053eb:

This was a temporary test used to check that timestamped
forks work, but now we can test it using the actual chain
specs for e.g. mainnet instead.

Reply:
Re-added the test to unblock